### PR TITLE
Modified rules to remove incorrect error call and provide more informative system status at rule firing; modified code for fast…

### PR DESCRIPTION
**Pull request by Gregg140**
…er execution, clarity, and correct processing of first ACISPKT

Also refactored the code to create "run_one_command" to remove code repetition; simplified last state maintenance..

Tested with  JUL3018 load (old power commands used)

                      .... and all the loads after JUL3018 all of which use the new               WSPOW000/WSPOW02A paradigm

                      AUG0218
                      AUG0618
                      AUG0718
                      AUG1318

  Plus the MAY0718_test load which came from the FOT and the MAR0518_ERRORS load which I hand edited to introduce errors.

### CODE REVIEW

**Review by richardjedgar — APPROVED** [review]
I'm happy with it since it passes the tests.

File: CHECK_POWER_COMMANDS/Backstop_File_Class.py [hunk 1]
```diff
@@ -91,9 +92,8 @@ def strip_out_ACISPKTs(self, backstop_file):
                                                    cmd) ],
                                                dtype = self.ACISPKT_dtype) ]
         
-            # Next check if the line is one of the Perigee Passage indicators
-            # NOTE: there will either be zero or one Perigee Passage Indicator in the line
-            if any(pp_ind in eachline for pp_ind in pp_indicators):
```
Comment (jzuhone) [inline CHECK_POWER_COMMANDS/Backstop_File_Class.py]: I feel like the old way is more readable, to be honest.

**Review by jzuhone — APPROVED** [review]
Other than the minor nit, sure

**Review by richardjedgar — APPROVED** [review]
Looks OK to me.

**Pull request merged**